### PR TITLE
Release 3.2.2

### PR DESCRIPTION
**Pull request by msusol**
* Fix browser caching of cdn.jsdelivr files.
* update ldavis.js to match ldavis.3.0.0.js

### CODE REVIEW

**Comment by msusol** [comment]
@bmabey Do you have a moment to merge this fix please?

FYI.. if you are interested in co-owners who can help, we can chat offline about the possibility.

File: pyLDAvis/urls.py [hunk 1]
```diff
@@ -16,26 +16,23 @@
 LOCAL_JS_DIR = os.path.join(__path__[0], "js")
 D3_LOCAL = os.path.join(LOCAL_JS_DIR, "d3.v5.min.js")
 
+# Avoid browser caching with @version in the URL.
```
Comment (bmabey) [inline pyLDAvis/urls.py:19]: This is somewhat unfortunate but probably fine.

**Review by bmabey — APPROVED** [review]
Great, thanks again for the fix and clean PR!

**Comment by bmabey** [comment]
I just released `v3.2.2`. I'd love help on the project, as you can tell from the commit history / releases this hasn't been top priority.

**Pull request merged**